handbook/engineering/distribution/onboarding.md

```diff
@@ -0,0 +1,81 @@
+# Distribution Engineer Oboarding
+
+Welcome to the distribution team! This document will guide you through distribution-specific onboarding tasks to make your start here at Sourcegraph as smooth as it can be.
+
+## Distribution Onboarding Checklist
+
+### Week 1
+
+#### Goals:
+- Clear knowledge of what the company’s, and Distributions’s mission, vision, values and culture are.
+- Know the expected outcome of each Distribution onboarding milestone.
+- Become familiar with Distribution [product](product.md),[tech stack](tech_stack.md) and [processes](recurring_processes.md).
+- Go through [General Onboarding](../../people-ops/onboarding/general_onboarding.md)
+- You have a sense of what tools you will be using and for what
+
+
+#### Tasks:
+- Meet your onboarding buddy
+- Add shared [Distribution calendar](https://calendar.google.com/calendar/u/0?cid=Y19qZHU3NTJ2anFpZ2NnMWVmYmYxZXIzY291Z0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) to your calendar
+- Attend weekly sync [meeting](https://calendar.google.com/event?action=TEMPLATE&tmeid=OXFyZXRsc21hdDh1MHNkNzQ2aGhqMzg3NmxfMjAyMTA1MjRUMTczMDAwWiBkYXZlQHNvdXJjZWdyYXBoLmNvbQ&tmsrc=dave%40sourcegraph.com&scp=ALL)
+- Deploy your own SG instance using one of the 3 deployment methods
```

Comment:
I would clarify this to created a personal GCP project to deploy into. 
Also, thoughts on doing docker-compose first then k8s?

Reply:
Add links too :)